homeassistant/components/freebox/sensor.py

```diff
@@ -11,10 +12,52 @@
 async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
     """Set up the sensors."""
     fbx = hass.data[DATA_FREEBOX]
-    async_add_entities([FbxRXSensor(fbx), FbxTXSensor(fbx)], True)
+    sys_config = await fbx.system.get_config()
+    fbx_sensors = []
+    for temp_sensor_id in sys_config["sensors"]:
+        fbx_sensors.append(FbxTempSensor(fbx, temp_sensor_id["id"]))
 
+    fbx_sensors.extend([FbxRXSensor(fbx), FbxTXSensor(fbx)])
+    async_add_entities(fbx_sensors, True)
 
-class FbxSensor(Entity):
+
+class FbxTempSensor(Entity):
```

Comment:
This entity should probably implement the `device_class` property and return `DEVICE_CLASS_TEMPERATURE`.